.github/workflows/build_wheels_windows.yml

```
@@ -31,6 +31,10 @@ on:
         description: "Script to setup environment variables for the build"
         default: ""
         type: string
+      build-params:
```

Please either change the name or description (if the same param will be passed to both wheel and conda build scripts)
```suggestion
      wheel-build-params:
```